src/css/components/filters.css

```
@@ -52,20 +41,38 @@
     font-weight: bold;
     padding: calc(var(--base-spacing) * 3) calc(var(--base-spacing) * 3);
     margin: 0;
-}
-.filter__title .icon {
-    display: none;
+    cursor: pointer;
+    user-select: none;
 }
 .filter__values {
     overflow-y: auto;
     padding:  0 calc(var(--base-spacing) * 3) 0 calc(var(--base-spacing) * 6);
     flex: 1;
+    max-height: 0;
+    transition: max-height .2s ease-in-out;
+    overflow: hidden;
+}
+details[open] {
+    height: 100%;
+}
+details[open] .filter__values {
     max-height: 11em;
```

Etant donné qu'il y a un max-height, il faut ajouter le `overflow-y: auto` ici.

```suggestion
     max-height: 11em;
     overflow-y: auto;
```